apps/omg_db/lib/omg_db/rocksdb/server.ex

```diff
@@ -115,14 +107,25 @@ defmodule OMG.DB.RocksDB.Server do
     do_get_single_value(parameter, state)
   end
 
-  def handle_call({:exit_info, utxo_pos}, _from, state) do
-    do_exit_info(utxo_pos, state)
-  end
-
   def handle_call({:spent_blknum, utxo_pos}, _from, state) do
     do_spent_blknum(utxo_pos, state)
   end
 
+  def handle_call({:get, type, specific_key}, _from, state) do
```

Comment:
We could have implemented all other specific value queries by this. It feels to me like design issue as we expose generic fetch function, also accepts a `type` which is internal detail (prefix) - so it assumes client of this function will provide correct value.

Reply:
The client of this would be the `Model`, so it should know what exact `type`(prefix) and the `specific_key` it is using as that is why there is such model.
Explained a bit on my motivation of exposing generic functions in previous comment.